pkg/ocm/helpers.go

```diff
@@ -357,7 +358,17 @@ func (c *Client) LinkOrgToRole(orgID string, roleARN string) (bool, error) {
 	return true, nil
 }
 
-func (c *Client) GetLinkedRoles(orgID string) ([]string, error) {
+func (c *Client) GetAccountLinkedRoles(accountID string) ([]string, error) {
+	resp, err := c.ocm.AccountsMgmt().V1().Accounts().Account(accountID).
+		Labels().Labels(USERRoleLabel).Get().Send()
+	if err != nil && resp.Status() != http.StatusNotFound {
+		return nil, handleErr(resp.Error(), err)
+	}
+
+	return strings.Split(resp.Body().Value(), ","), nil
+}
+
+func (c *Client) GetOrganizationLinkedRoles(orgID string) ([]string, error) {
```

Comment:
If the label is hardcoded inside the func I would rename the func to `GetOrganizationLinkedOCMRoles` so it's more verbose.